lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,16 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'id' => $data->field_northstar_id_value,
```

Comment:
oh, well I think we want this to be `null` though if there is no id and not `"NONE"`... maybe don't use the `dosomething_helpers_isset()` and just use a simple ternary that turns "NONE" to `null`.